packages/flutter/lib/src/animation/listener_helpers.dart

```diff
@@ -30,6 +31,7 @@ abstract class LazyListenerMixin implements _ListenerMixin {
   bool get isListening => _listenerCounter > 0;
 }
 
+/// A mixin that helps listen to another object even when this object does not have registered listeners.
```

Comment:
It doesn't really help anything. It just replaces the didRegisterListener/didUnregisterListener contract with a dispose contract.